### ISSUE TITLE
Uncaught Error: Unsupported Bridge operation "0"

### ISSUE THREAD

**Issue by msageryd** [facebook/flipper#3197]
## 🐛 Bug Report

As soon as I go into React DevTools in Flipper, the app restarts and I get the following error in Flipper.
![image](https://user-images.githubusercontent.com/4483085/146743719-8d3dbfdc-a2de-4311-9bde-a4475b88f025.png)

The other plugins (Hermes, Logs, Layout, etc) works as they should, even after the restart.

Edit:
I just discovered another error in the log, just before the crash:
![image](https://user-images.githubusercontent.com/4483085/146746762-d6ca2609-a369-49b8-98e5-89ac1e708628.png)

## Environment

React-Native: 0.66.3
Flipper 0.126
XCode: 13.1
MacOS: 12.0.1 on Intel

**Comment by 000xuandu** [comment]
Same issue

**Comment by mweststrate** [comment]
This might be an duplicate of #3189, not fully sure. Does the workaround listed here, https://github.com/facebook/flipper/issues/3189#issuecomment-997191720, or upgrading to 0.127 solve the issue?

**Comment by msageryd** [comment]
Thanks.
I had already applied your manual patch according to #3189, this did not fix the current problem.

I have now upgraded to 0.127.0 and changed my Podfile to `use_flipper!({'Flipper' => '0.127.0'})`. Pod install fails due to incompatible "FlipperKit/FlipperKitNetworkPlugin". How should 0.127.0 be installed correctly?

**Comment by msageryd** [comment]
Update..
I'm running NVM. Could this have something to do with the Devtools mess? I did a search for react-devtools and got this:
![image](https://user-images.githubusercontent.com/4483085/147461856-4149aceb-c9f9-4f8c-8923-3ffbcb24f0bb.png)

My current Node version is 14.16.1. Could Flipper find Devtools from another version? Is there anywhere I can see where Flipper finds react-devtools?

**Comment by msageryd** [comment]
Solved at last!

tl;dr
```
npm i -d react-devtools-core@latest --save-dev
npm i -g react-devtools@latest
``` 

I did not have **react-devtools-core** at all in my package.json.

I don't know why I need react-devtools-core (everything worked fine with Chrome debugger and react-native-debugger before)

I have not read anything about react-devtools-core in the Flipper docs. Did I miss something?

**Comment by kofkgoing** [comment]
```
"react-devtools-core": "4.24.3",
  "resolutions": {
"react-devtools-core": "4.24.3"
},
```

https://github.com/facebook/flipper/issues/3649?fbclid=IwAR3wycD10ojRA2v3alwuaNeykbVzjnfiYkqsqHJX5Fj6YKj_QK0TtR1Fm7M#issuecomment-1272444843
